example/migrate/kube-scheduler.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: v1
```

Comment:
I believe you can delete the entire `example/migrate` directory because it is no longer used.

Reply:
Sorry, deleted `example/migrate` directory too.